lib/ex_crypto.ex

```diff
@@ -36,6 +36,33 @@ defmodule ExCrypto do
     {kind, Exception.normalize(kind, error), System.stacktrace}
   end
 
+  @aes_block_size 16
+  @iv_bit_length 128
+
+  defp key_and_iv_for_algorithm(raw_key, raw_iv, algorithm) do
```

Comment:
Oh good point, hadn't thought of hex encoding, not seen that done... Plus there's URL safe base64...